__tests__/MyComponentAccessibility.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { axe, toHaveNoViolations } from '@axe-core/react';
+
+// Adding the toHaveNoViolations matcher
+expect.extend(toHaveNoViolations);

```

Comment:
It might make more sense to move this to the setup-tests file?

Reply:
done @lindapaiste